app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -262,6 +270,12 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.note = $scope.realmNote;
     } else if ($scope.emsCommonModel.emstype === 'rhevm'){
       $scope.emsCommonModel.metrics_api_port = "5432";
+    } else if ($scope.emsCommonModel.emstype === 'openshift' || $scope.emsCommonModel.emstype === 'openshift_enterprise'){
+      $scope.emsCommonModel.default_api_port = "5000";
+    } else if ($scope.emsCommonModel.emstype === 'atomic' || $scope.emsCommonModel.emstype === 'atomic_enterprise'){
+      $scope.emsCommonModel.default_api_port = "5000";
+    } else if ($scope.emsCommonModel.emstype === 'kubernetes'){
+      $scope.emsCommonModel.default_api_port = "5000";
```

Comment:
@yaacov how come you changed these from `8443` to `5000`? The default port for the API endpoint is `8443`.